### ISSUE TITLE
confuse about the coordinate

### ISSUE THREAD

**Issue by chongtianyifa** [VirtualBrainLab/Pinpoint#363]
Hi, 
I noticed that there is a huge difference in coordinate between Pinpoint and other software. Taking the center of VTA as an example, its AP is around 3.2 mm relative to bregma in Andy Peters’ trajectory explorer or Paxinos Atlas. In contrast, the AP is around 2.6 mm relative to bregma after MRI transformation in Pinpoint. In my experience, 3.2 mm is the right target. Do you have some explanation about the discrepancy?

Thanks,
Chris

**Comment by dbirman** [comment]
Hi Chris -- that's definitely a big gap between the other atlases and Pinpoint. The difference could come from where we're defining Bregma, although at some point Andy and I coordinated on that. Do you know the Bregma coordinate in trajectory explorer or Paxinos?

This is the right place for issues, thanks.

**Comment by dbirman** [comment]
One thing that may be helpful to do is to not use the MRI transform. You can turn it off in the Atlas settings (see image) by switching to "CCF Atlas". You'll find that VTA is at ~3mm from Bregma when you make that switch.

![image](https://github.com/VirtualBrainLab/Pinpoint/assets/7455770/951331eb-1645-4e27-9ca8-a1b43dc67f2c)

**Comment by chongtianyifa** [comment]
Hi,
According to Andy's neuropixels_trajectory_explorer script, bregma was estimated at [540,44,570] in 10 um CCF atlas. I think it matches Pinpoint right now：
![image](https://github.com/VirtualBrainLab/Pinpoint/assets/46367566/549f80ed-8490-4838-a20d-4e374f3cb0b2)
Hope you can figure out how to narrow the gap in the future so that we can use Pinpoint more confidently.

Thanks,
Chris

**Comment by dbirman** [comment]
I just tried to quickly line up VTA in Andy's viewer and I get an AP of 3 relative to Bregma. Can you show me how you're getting an AP of 3.2? 
<img width="760" alt="Screen Shot 2023-06-15 at 8 03 36 PM" src="https://github.com/VirtualBrainLab/Pinpoint/assets/7455770/8e68339e-f33b-425e-9115-b72887028c89">

I also tried looking at the Allen Institute viewer to double check that there isn't an issue. In raw CCF coordinates in Pinpoint (reference coordinate set to 0,0,0) the right hemisphere center is at ap=8.35, ml=6.5, dv=5, which pretty much exactly matches what I see in the Allen Institute's viewer: http://connectivity.brain-map.org/3d-viewer?v=1&types=PLY&PLY=749 where the center is ~8.3ish, 6.5, 5. For reference here's a screenshot of the rough "center" of VTA in the allen viewer:
<img width="1219" alt="Screen Shot 2023-06-15 at 7 58 15 PM" src="https://github.com/VirtualBrainLab/Pinpoint/assets/7455770/03678b30-ddbd-41df-a9a4-28875c09fae7">

If you can share some more information on how you're defining VTA in each system I think we can track down where the discrepancy is coming from, but so far I'm having trouble re-creating it this. 

Also a little more context: The MRI Transform is based on a particular average of mouse brains, but if this average doesn't align well with your experience you can turn it off and plan your insertions in raw CCF space. We also have the Paxinos space that will be available soon as an option as well. You can read more about the current transform options here: https://virtualbrainlab.org/pinpoint/in_vivo_alignment.html

**Comment by chongtianyifa** [comment]
I just compared the coordinate of VTA in Andy Peters’ trajectory explorer and Pinpoint more carefully. I put the coordinate of VTA center output from Pinpoint:
![image](https://github.com/VirtualBrainLab/Pinpoint/assets/46367566/5f6c4fef-3809-47cb-be06-057e248a8a76)
into Andy's trajectory explorer, 
![image](https://github.com/VirtualBrainLab/Pinpoint/assets/46367566/3a9e510d-a64e-45cc-b5f2-9d4a4c9645e0)
I think it just hit the very anterior surface of VTA. I have to say that this comparison might not be fair since the transformations might be different.
According to Paxions altlas, AP of VTA ranges from -2.8 to -3.8.

**Comment by dbirman** [comment]
Is it possible you're on a different version of the trajectory explorer? I downloaded the latest version today and copying your coordinates I get what's below, which is consistent with Pinpoint. Is the version of Andy's explorer you're using the latest version from the website?

<img width="1167" alt="Screen Shot 2023-06-15 at 10 22 49 PM" src="https://github.com/VirtualBrainLab/Pinpoint/assets/7455770/80859fe6-2017-4ba9-85a5-55aa9397edc4">

As far as Paxinos, we're going to add it to Pinpoint -- hopefully in a few weeks. As far as I know the Paxinos atlas is larger than the CCF, so you can get the same effect as using the Paxinos atlas by using the bregma-lambda scaling feature in Andy's explorer or in Pinpoint and using a B-L distance > 4.1 (this feature isn't on the web right now, but it's on the development build for Pinpoint, which I can give you access to if you want to try that)

**Comment by chongtianyifa** [comment]
Yes, you are right, We are using different version of trajectory explorer.
Newest:
![image](https://github.com/VirtualBrainLab/Pinpoint/assets/46367566/0b3f0752-2d93-4369-8b9c-ec38b1955fe4)
My version:
![image](https://github.com/VirtualBrainLab/Pinpoint/assets/46367566/9b1c1e97-ed29-49d1-8fb7-9bfc0070b4da)
It will be great if I can try Paxinos atlas version.

**Comment by dbirman** [comment]
Great, glad we tracked that down. I'll try to send an update when we have Paxinos added, aiming for that sometime in July.
